src/main/views/form/main/template.njk

```diff
@@ -64,7 +40,7 @@
 
   {% if "claim-employment-tribunals" in currentHost %}
     <!-- Dynatrace RUM Script -->
-      <script type="text/javascript" src="https://js-cdn.dynatrace.com/jstag/17177a07246/bf00910jpo/15d45f7c5933b5bc_complete.js"crossorigin="anonymous"></script>
+      <script type="text/javascript" src="https://js-cdn.dynatrace.com/jstag/17177a07246/bf00910jpo/15d45f7c5933b5bc_complete.js" crossorigin="anonymous"></script>
```

Comment:
Can this be removed from the code base as it's for the ET1 frontend as opposed to ET3

Reply:
Removed traces of ET1 Dynatrace from template